x-pack/plugins/security_solution/public/management/cypress/e2e/rbac/endpoint_role_rbac_with_space_awareness.cy.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login, ROLE } from '../../tasks/login';
+import { createSpace, deleteSpace } from '../../tasks/spaces';
+import {
+  clickEndpointSubFeaturePrivilegesCustomization,
+  clickFlyoutAddKibanaPrivilegeButton,
+  clickRoleSaveButton,
+  clickViewPrivilegeSummaryButton,
+  expandEndpointSecurityFeaturePrivileges,
+  expandSecuritySolutionCategoryKibanaPrivileges,
+  navigateToRolePage,
+  openKibanaFeaturePrivilegesFlyout,
+  setEndpointSubFeaturePrivilege,
+  setKibanaPrivilegeSpace,
+  setRoleName,
+  setSecuritySolutionEndpointGroupPrivilege,
+} from '../../screens/stack_management/role_page';
+
+describe(
+  'When defining a kibana role for Endpoint security access with space awareness enabled',
+  {
+    tags: ['@ess', '@serverless', '@serverlessMKI'],
+    env: {
+      ftrConfig: {
+        productTypes: [{ product_line: 'security', product_tier: 'essentials' }],
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'endpointManagementSpaceAwarenessEnabled',
+          ])}`,
+          // Enable spaces for serverless
+          `--xpack.spaces.maxSpaces=10`,
```

Comment:
don't fleet experimental features `subfeaturePrivileges` and `useSpaceAwareness` needed to be enabled for space awareness? (mentioned in `x-pack/plugins/security_solution/common/experimental_features.ts`)

Reply:
yes... and no 😄 

Once we get around to actually start to implement space awareness into our application code, then yes, we'll need to ensure the fleet feature is enabled (and manual trigger migration)... but for testing this specific PR/change to feature controls, we don't need it.

Reply:
haha, i see